tsconfig.json

```diff
@@ -15,7 +15,9 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "outDir": "dist",
-    "sourceMap": true
+    "sourceMap": true,
+    "allowUnusedLabels": false,
```

Comment:
Isn't this already the default? https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
After review, we have decided to remove this all well.